companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
+        "it",
+        "us",
+        "spa",
+        "br",
+        "fr"
+    ],
+    "categories": [
+        "it",
+        "commerce"
+    ],
+    "name": "Zucchetti",
```

Comment:
Please use the full legal name of the company here. In this case, this seems to be `Zucchetti s.p.a.`.